docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## Description
+
+<!--- A brief description of this PR and the problem is trying to solve -->
+
+## Linked issue
+
+<!--- Put here the relevant issue from YouTrack -->
+
+## Type of change
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
+- [ ] 🛠 New feature (non-breaking change which adds functionality)
+- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] ⛑ git-flow chore (backport, hotfix, etc)
+
+## Developer checklist
+<!--- A mental checklist for the developer submitting the PR --->
+- [ ] I have read the [style guide](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/style-guide.md) document, and my code follows the code style of this project.
+- [ ] If my code deals with exceptions, it follows the [guidelines](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/exceptions.md).
+- [ ] I have updated any documentation accordingly, if needed.
```

Comment:
It's a dream! Awesome dream, but (almost) nobody will do it... Unfortunately...

Reply:
If they don't, we won't merge -- that's as simple 😁

Reply:
I like it - but in real world there are use cases where we don't have time for some checks on this list (for example if we are rushing to some deadline, there is no time to write proper tests). But it should fit most of the time

Reply:
> there is no time to write proper tests

I think we should try to move away from this mentality as much as possible, otherwise Anatoli might come to your house with a baseball's bat :trollface:

Reply:
:D I agree. Its just sentiment that I got so far on the project

Reply:
The sentiment is ours to change :smile:

Reply:
I can see the use in a section "If you didn't implement tests or documentation, why not?" section, potentially, but I wouldn't want to encourage that.